include/zephyr/bluetooth/l2cap.h

```diff
@@ -83,6 +84,28 @@ extern "C" {
  */
 #define BT_L2CAP_SDU_BUF_SIZE(mtu) BT_L2CAP_BUF_SIZE(BT_L2CAP_SDU_HDR_SIZE + (mtu))
 
+/** @brief L2CAP ECRED minimum MTU
+ *
+ *  The minimum MTU for an L2CAP Enhanced Credit Based Connection.
+ *
+ *  This requirement is inferred from text in Core 3.A.4.25 v6.0:
+ *
+ *      L2CAP implementations shall support a minimum MTU size of 64
+ *      octets for these channels.
+ */
+#define L2CAP_ECRED_MIN_MTU 64
+
+/** @brief L2CAP ECRED minimum MPS
+ *
+ *  The minimum MPS for an L2CAP Enhanced Credit Based Connection.
+ *
+ *  This requirement is inferred from text in Core 3.A.4.25 v6.0:
+ *
+ *      L2CAP implementations shall support a minimum MPS of 64 and may
+ *      support an MPS up to 65533 octets for these channels.
```

Comment:
Curious; the maximum is `65533` which is not the same at UINT16_MAX. Suggest to also add a `MAX` value

Reply:
I was considering adding the max value define here but decided against. It would be useless in our system since we instead have a implementation defined limit `BT_L2CAP_RX_MTU` on MPS. `BT_L2CAP_RX_MTU` is limited to `65533` by kconfig.